### PR TITLE
PR for a wonderful readme.md 

### PR DESCRIPTION
**Pull request by FrostyTheDumDum**
Added a Readme.MD cause why the fuck not lol

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+![Header](https://github.com/FrostyTheDumDum/Gap_Utilities/blob/master/header.png?raw=true "Header")
+
+# Eureka!
+The Official Repository Of The Eureka Bot! :eyes:
+
+
+https://discordapp.com/oauth2/authorize?client_id=791700965081612288&scope=bot&permissions=8 ~~gimme admin lol~~
```
Comment (nearlySplat) [inline README.md:7]: I think the README should be about the code, how to contribute and not the Ad. Sorry :(

**Review by nearlySplat — CHANGES_REQUESTED** [review]
I also feel like the image is a bit bland.

File: README.md (hunk 1 above)
Comment (FrostyTheDumDum) [reply README.md:7]: Lol what???????????????????????????????????????????????????? :(

**Comment by nearlySplat** [comment]
@FrostyTheDumDum should you ever feel like re-opening this, do, but for now, I will keep it closed. I don't think a README is something really needed at the moment.

**Pull request closed without merging**